### DISCUSSION TITLE
image는 내부에 위치할 수 없나요?

### DISCUSSION THREAD

**Discussion by thebbong** [hyesungoh/comet-land#73] [Q&A] [answered]
외부 이미지가 아닌 블로그 내에 이미지를 어떤 path에 둬야하는지 궁금합니다.

**Comment by hyesungoh** [comment, marked as answer]
어디에서 사용되는 이미지일까요?

[해당 경로](https://github.com/hyesungoh/comet-land/tree/03e1fa132ce61c71af6444c7c1c509c600c8d773/packages/core/constants/General)에서 사용되는 이미지 외에, 추가적인 이미지를 사용하시고 싶으시면

각 앱의 public 폴더에 넣으셔서 사용하시면 될 것 같습니다.

- `apps/blog/public`
- `apps/resume/public`

아실 수도 있으시지만 참고 링크 공유드리겠습니다.

https://nextjs.org/docs/basic-features/static-file-serving

**Reply by thebbong** [reply]
👍
